cli.sh

```diff
@@ -779,7 +852,7 @@ execute_che_mount() {
 
   # If extra parameter provided, then this is the port to connect to
   if [ $# -eq 1 ]; then
-    info "${CHE_MINI_PRODUCT_NAME} mount: Connecting to remote workspace on port ${1}"
+    info "${CHE_MINI_PRODUCT_NAME} mount: Connecting to workspace on port ${1}"
```

Comment:
I simplified things.  I have two modes for INFO now.  for certain outputs like `che info` where there is a lot of streming output, I do not print out any additional information other than `INFO:`.  For others, I print out `($CHE_MINI_PRODUCT_NAME <command-name>): <statement>`